README.md

```diff
@@ -57,9 +57,48 @@ This driver supports only one topology key:
 that represents availability by zone.
 
 ## Kubernetes User Guide
-
 ### Install Driver
 
+#### Preflight (Ubuntu like)
+1. You need to install golang
+```
+sudo apt update && sudo apt install golang
+```
+2. Set path for go
+You can read [here](https://github.com/golang/go/wiki/SettingGOPATH) more or quick run this steps
+Set path to /home/$USER/go (my user is de1m)
+```
+export GOPATH=$HOME/go
+```
+3. Clone the repo to path below
+```
+mkdir -p $GOPATH/src/sigs.k8s.io
+cd $GOPATH/src/sigs.k8s.io
+git clone https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver.git
+```
+4. Now you need to set the project variables
+-   **PROJECT** - is the id of your project in gcp. Show below, you need the PROJECT_ID
+```
+de1m@comp0:~$ gcloud projects list
```

Comment:
please remove all mentions of your username. Just do
`$ gcloud projects list` instead